Twilio/Tests/Unit/TwiML/MessagingResponseTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Twilio\Tests\Unit\TwiML;
+
+use DOMDocument;
+use SimpleXMLElement;
```

Comment:
This PR switches away from SimpleXML, right?

Reply:
oops, yes we should just use `DOMDocument` everywhere